interpreter.asm

```diff
@@ -114,7 +110,7 @@ interpret_loop
     rts
 
 interpret_tib
-    jsr	INTERPRET
+    jsr INTERPRET
```

Comment:
I think there will need to be a check that detects if code space and header space have grown so big that they are threatening to overwrite each other. It might be that this is the best place to put such a check. I am not sure how big the safety margin should be.

Reply:
Also, what code formatting should be in play here?  I thought it was expanded tabs = 4 spaces, but I guess that just happened to align when I retabbed the file.

Reply:
It is a good question. I also think it should be expanded tabs=4 spaces. I have not followed that consistently in the past. I could make an issue to add an [.editorconfig](https://editorconfig.org/) file for clarity.

Reply:
So, if I do a check for unused space right here, won't a full dictionary result in a deadlocked interpreter?

Reply:
I think it would make sense to print warnings when there is, for example, less than 256 bytes left.

Reply:
Doing it right there will be either a lot of code or will use some stack space, requiring the stack check to be shallower, I think.